### PR TITLE
Bump march hare to 2.11.0

### PR DESCRIPTION
**Pull request by andrewvc**
Bump march_hare version to match new rabbitmq output plugin march_hare version

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* 1.1.0
+  - Bump march hare version to 1.1.0
```
Comment (michaelklishin) [inline CHANGELOG.md]: Do you mean `2.11.0`?

Comment (andrewvc) [inline CHANGELOG.md]: Ack, yes :) Good catch

**Comment by ph** [comment]
code LGTM, havent tested with a real rabbitmq. We should have some test for this plugin :(

**Comment by andrewvc** [comment]
@ph I would too! I added a bunch to the output plugin, when I have some time I'll tackle this one.

**Pull request closed without merging**